.github/workflows/test.yaml

```diff
@@ -160,7 +160,7 @@ jobs:
         shell: bash
         run: |
           brew install colima docker
-          colima start --mount $HOME/.kwok/:w
+          colima start --mount $HOME/.kwok/:w --mount $(pwd):w --mount-type virtiofs
```

Comment:
The reason Mac workflows were failing... There was an [issue](https://github.com/abiosoft/colima/issues/503) in colima with latest mac causing this problem